cross/libffi/patches/includesdir.patch

```diff
@@ -1,10 +1,25 @@
-diff -ur include-old/Makefile.am include/Makefile.am
---- include-old/Makefile.am	2011-08-22 20:38:12.000000000 +0200
-+++ include/Makefile.am	2011-12-16 23:15:45.000000000 +0100
-@@ -5,5 +5,5 @@
- DISTCLEANFILES=ffitarget.h
- EXTRA_DIST=ffi.h.in ffi_common.h
+From 982b89c01aca99c7bc229914fc1521f96930919b Mon Sep 17 00:00:00 2001
```

Comment:
To be honest I could move this outside of this PR since it turns out to be unrelated and not required.